### PR TITLE
Revert "fix(shopify): remove not implemented feature text"

### PR DESCRIPTION
**Pull request by cbernat**
This reverts commit 88d73815ab2419a5daefd6b6459a2b4eb113cff3.

Pending to be merged when all features are in place.

### CODE REVIEW

File: src/i18n/es.js [hunk 1]
```diff
@@ -179,8 +179,8 @@ export default {
     error_cannot_access_api: `Ouch! No pudimos conectar con la API de Shopify, por favor vuelve a intentarlo luego.`,
     header_disconnected_warning: `Al presionar "Conectar" serás redirigido a Shopify, donde podrás realizar todos los pasos necesarios para integrar.`,
     header_store: `Nombre de la cuenta:`,
-    header_subtitle: `Envía automáticamente los Contactos de tu tienda y toda su información a una Lista de Doppler. Además, podrás crear Automations
-    de Carrito Abandonado y Producto Visitado. ¿Tienes dudas? Presiona <a target="_blank" href="${urlHelp}/como-integrar-doppler-y-shopify/" ${patchForBlank}>HELP</a>.`,
+    header_subtitle: `Envía automáticamente los Contactos de tu tienda y toda su información a una Lista de Doppler. Además, podrás importar los productos de tu tienda en
+    Plantillas de Email y crear Automations de Carrito Abandonado y Producto Visitado. ¿Tienes dudas? Presiona <a target="_blank" href="${urlHelp}/como-integrar-doppler-y-shopify/" ${patchForBlank}>HELP</a>.`,
```
Comment (fcoronelmakingsense) [inline src/i18n/es.js]: why space between 'en' and 'plantillas' ?

Comment (cbernat) [reply src/i18n/es.js]: I see no space there. What do you mean?

Comment (fcoronelmakingsense) [reply src/i18n/es.js]: my bad, in other mode view see and space

**Pull request merged**